diff --git a/node_modules/expo-modules-core/android/CMakeLists.txt b/node_modules/expo-modules-core/android/CMakeLists.txt
index 1111111..2222222 100644
--- a/node_modules/expo-modules-core/android/CMakeLists.txt
+++ b/node_modules/expo-modules-core/android/CMakeLists.txt
@@ -134,4 +134,5 @@ target_link_libraries(
 target_link_libraries(
   ${PACKAGE_NAME}
   ReactAndroid::reactnative
+  c++_shared
 )
